content/rfcs/_index.md

```diff
@@ -2,7 +2,7 @@
 title: "RFCs"
 description: "This page lists all the RFCs which define major works regarding Gaia."
 weight: 10
-alwaysopen: true
+alwaysopen: false
```

Comment:
Ops. :))) my bad. :)

Reply:
No problem. Didn't saw it too during review. 🤗